CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - `StrictNoSign` enabled.
 - Added JSON-RPC Private API to enable using symmetric or asymmetric cryptography to encrypt/decrypt message payloads
 - Refactor: Move `waku_filter` protocol into its own module.
+- Refactor: Split out `waku_types` types into right place; create util folder.
```

Comment:
This probably go to "new" Next Version now, but likely to show up as merge conflict in any case.

Reply:
Done